pyomo/core/kernel/__init__.py

```diff
@@ -59,20 +59,22 @@
 from pyomo.core.expr.calculus.derivatives import differentiate
 from pyomo.core.expr.taylor_series import taylor_series_expansion
 
-import pyomo.core.kernel.base
-import pyomo.core.kernel.homogeneous_container
-import pyomo.core.kernel.heterogeneous_container
-import pyomo.core.kernel.variable
-import pyomo.core.kernel.constraint
-import pyomo.core.kernel.matrix_constraint
-import pyomo.core.kernel.parameter
-import pyomo.core.kernel.expression
-import pyomo.core.kernel.objective
-import pyomo.core.kernel.sos
-import pyomo.core.kernel.suffix
-import pyomo.core.kernel.block
-import pyomo.core.kernel.piecewise_library
-import pyomo.core.kernel.set_types
+from . import (
```

Comment:
I again register my relative path complaint. And now that it's for the third time I will express why:

- IDE and Tooling Support: Many IDEs and static analysis tools have better support for resolving absolute imports.
- Flexibility and Scalability: Absolute imports remain stable and clear regardless of how deep the package structure goes.